soccer/include/rj_vision_filter/vision_filter.hpp

```diff
@@ -139,14 +138,13 @@ class VisionFilter : public rclcpp::Node {
     rclcpp::TimerBase::SharedPtr predict_timer_;
 
     rclcpp::Subscription<DetectionFrameMsg>::SharedPtr detection_frame_sub_;
-    rj_utils::ConcurrentQueue<DetectionFrameMsg::UniquePtr>
-        detection_frame_queue_;
+    rj_utils::ConcurrentQueue<DetectionFrameMsg::UniquePtr> detection_frame_queue_;
 
     /**
      * @brief Publisher for WorldStateMsg.
      */
     rclcpp::Publisher<WorldStateMsg>::SharedPtr world_state_pub_;
 
-    params::ROS2ParamProvider param_provider_;
+    ::params::ROS2ParamProvider param_provider_;
```

Comment:
Should this just be renamed to `rj_params`?

Reply:
lol rip I guess this is for another PR then.